doc/processingsteps.rst

```diff
@@ -363,6 +383,25 @@ The *Normals* option colors the mesh by surface normal direction, and the *NbPro
    A fused mesh colored by the number of views
    that see each part of the surface.
 
+Additional check boxes are available to enable removal of color
+on some surfaces that should not be colored.
+The *Remove masked surface color* option uses a mask video loaded
+into TeleSculptor and does not apply color at points under this mask.
```

Comment:
This command can be used to hide the rough border of the generated mesh.

Reply:
I think this statement is specific to the AP Hill sample data which has darker or black pixel at the image border.